src/Common/src/System/Security/Cryptography/Asn1/asn.xslt

```diff
@@ -57,8 +57,7 @@ namespace <xsl:value-of select="@namespace" />
         static <xsl:value-of select="@name" />()
         {
             <xsl:value-of select="@name" /> decoded = default;
-            AsnReader reader;<xsl:if test="asn:SequenceOf | asn:SetOf">
-            AsnReader collectionReader;</xsl:if><xsl:apply-templates mode="DefaultFieldVerify" />
```

Comment:
Instead of deleted, this should probably just change the if to `asn:SequenceOf[@defaultDerInit] | asn:SetOf[@defaultDerInit]`.

It would probably fail to compile on something like (from X.680) `children SEQUENCE OF ChildInformation DEFAULT {}`

Or, from my testbed project:

```xml
  <asn:SetOf name="SetOfPrintableStringImplicitDefault" implicitTag="12" defaultDerInit="0xAC, 0x04, 0x13, 0x02, 0x34, 0x32">
    <asn:PrintableString/>
  </asn:SetOf>
```

Reply:
I don't know that there's a real use-case for it, but it's legal ASN.1, so it could come up at some point :smile:.